ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts

```diff
@@ -339,6 +340,27 @@ test.describe('Attachment Point Tool', () => {
     expect(molFile).toEqual(molFileExpected);
   });
 
+  test('Click and Save as *.mol file', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1747
+    Description: Click the 'Save As' button, and click the 'Save' button.
+    Open the saved *.mol file and edit it in any way.
+    */
+    await openFileAndAddToCanvas('chain-with-attachment-points.ket', page);
```

Comment:
I propose to add files to the corresponding folder under `test-data`

Reply:
Ok. I will do it.

Reply:
Done.